### DISCUSSION TITLE
Neey/Heltec 4A Bluetooth Balancer - Bluetooth support like the JK-BMS?

### DISCUSSION THREAD

**Discussion by moench13** [Louisvdw/dbus-serialbattery#756] [Ideas]
Hello,
it is possilbe to implement the Neey/Heltec Bluetooth Balancer like the JK-BMS over Bluetooth... many People use the 4A Smart Acitve Balancer so it is possilbe to use is for BMS Virtual Communication.
Good Idea?
greets from Austria - great Project!

**Comment by mr-manuel** [comment]
Unfortunately I have none. If someone has a spare one I can extend the script.

**Comment by Sailingben** [comment]
I like this, nice if it works with Enerkey/Neey 4amp en also there 8 and 10amp models
